projects/testing/core/tag.harness.ts

```diff
@@ -0,0 +1,40 @@
+import {TestElement, TestKey} from '@angular/cdk/testing';
+import {TuiComponentHarness} from '@taiga-ui/testing/utils';
+
+import {TuiSvgHarness} from './svg.harness';
+
+export class TuiTagHarness extends TuiComponentHarness {
+    static hostSelector = `tui-tag`;
+
+    async getCrossIcon(): Promise<TuiSvgHarness | null> {
+        const svg = await this.locatorForOptional(
+            TuiSvgHarness.with({selector: `.t-icon`}),
+        )();
+
+        return svg;
+    }
+
+    async getTagDiv(): Promise<TestElement> {
```

Comment:
This sounds a bit too low level/implementation detail, I wouldn't want to expose that. If it is only used for color test maybe we can test it some other way?

Reply:
providing just background color might be an option for now ?

Reply:
@waterplea can i have your opinion here.

Reply:
Yes, let's go with background color only. We are not publishing this package yet so we might have breaking changes in future if we need to.

Reply:
done ✅